### ISSUE TITLE
Turning Flint into flaked flint: fail if you have flaked flint in the inventory

### ISSUE THREAD

**Issue by keybounce** [DarkPacks/SevTech-Ages#2940]
## Bug Report

I am attempting to turn flint into flaked flint.
If I do not have flaked flint in my inventory and I am banging a rock near me, or if I am banging a rock at a distance (even if there is a pile of flaked flint on the ground), it works.

However, if I am banging a rock near me, and I have flaked flint in my inventory, then I do not get an additional flaked flint. The original flint is lost.

## Expected Behaviour

I should get a flaked flint every time.

## Steps to Reproduce (for bugs)

1. Break grass for fiber, then twine.
2. Break leaves for sticks.
3. Combine sticks and twine into a mesh
4. Break some gravel.
5. Use gravel and mesh to make flint (at least 2).
6. Find some exposed stone. Get close to it.
7. Break a flint against the stone. Get a flaked flint.
8. Break a 2nd flint against the stone. Lose it, get nothing.

Tested with a true server; not tested in single player.

## Logs

* Client/Server Log:
* Crash Log:

## Client Information

* Modpack Version: 3.0.8
* Java Version: 8
* Launcher Used: vnilla
* Memory Allocated: Server is using 1.5 GB; client is using 2.5 GB.

* Server/LAN/Single Player:
* Optifine Installed:
* Shaders Enabled:

## World Information

* Modpack Version world created in: 3.0.8

* Additional Content Installed:

## Server Information
* Java Version: 8
* Operating System: mac 10.9.5
* Hoster/Hosting Solution: me
* Sponge (Non-Vanilla Forge) Server:

**Comment by artdude543** [comment]
I've tested this over and over (from reports from Discord) and it's not an issue it works fine each time. Note: The item can be spat out anywhere and will fly off this is partly intended and a bug. But we've not pushed for this to be fixed on the mod side, nor will it happen anytime soon due to their own IRL work.

TLDR: Not an issue, tried much time and got the item each time both single and server worlds.

**Comment by keybounce** [comment]
Retested with Java 8, 171. 

First round of 12 flint gave 10 flakes; second round of 12 flint gave 9 flakes. Testing 9 flakes gave 8 points.

If it is supposed to be giving 100% drop, it is not. If it is supposed to be giving "most of the time" drop, it is working. With no documentation, I have no idea what is the expected/intended behavior.

If it is supposed to be 100%, I will be happy to help debug this.

The area being tested was cleared of tall grass and other distractions. After breaking all the flint, I went and hugged the wall just to make sure that nothing was hidden inside blocks. Testing was done in 2 different areas; one was a wide flat plain with a small wall, the other was on the side of a mountain after removing some dirt to get inside to where a rock area was.

**Comment by artdude543** [comment]
It's not 100% and it's random to the world tick.
